docs/web/contribute.md

```diff
@@ -151,82 +274,92 @@ In case you need to manually update a dependency see the next section.
 3. Run
 
 ```sh
+# if using nix
+$ ./dev/poetry2setup -o setup.py
+
+# it not using nix, requires installation of tomli and poetry-core
```

Comment:
these are in the conda files?

Reply:
Not explicitly, but they are both a `pip install`/`conda install` away.